docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/bias_quantification.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<p>As I mentioned in the last documentation session, I think the intervention bubble in this diagram is misleading because you could conclude that you are fine if the intervention does not affect <strong>both</strong> risks! I'd vote to remove the bubble altogether, with the implied understanding that R1 or R2 should be impacted by the intervention (directly or indirectly).</p><p> </p><p>I'm not sure I agree with the text here. It seems to imply that if not for the difference between PAF bias and intervention impact bias, we'd just accept the same limitation as GBD. But I don't think it's always true that we want to accept all of GBD's limitations: GBD is restricted in what it can do by making estimates for all years/locations/causes, while we try to make a higher-fidelity model with a much smaller scope.</p><p> </p><p>I think this section should call out that when this type of correlation <em>is</em> included in a model, the correlation-aware joint PAF calculation should be used, as described on the risk-risk correlation page.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>